library/std/src/sys/unix/time.rs

```diff
@@ -70,12 +64,24 @@ impl fmt::Debug for SystemTime {
 
 impl Timespec {
     pub const fn zero() -> Timespec {
-        Timespec::new(0, 0)
+        unsafe { Self::new_unchecked(0, 0) }
     }
 
-    const fn new(tv_sec: i64, tv_nsec: i64) -> Timespec {
+    const fn new_assert(tv_sec: i64, tv_nsec: i64) -> Self {
```

Comment:
Could we not just replace this with `new().unwrap()`?

Reply:
done